wormhole-connect/src/views/v2/Bridge/ReviewTransaction/GasSlider.tsx

```diff
@@ -117,7 +117,6 @@ const GasSlider = (props: {
     const tokenAmount = amount.display(
       amount.truncate(props.destinationGasDrop, 6),
     );
-    console.log(tokenAmount);
```

Comment:
This one was creating some noise in the console when in Review Transaction view. Please let me know if you think it'd still be useful when debugging.